exchanges/binance/binance_wrapper.go

```diff
@@ -935,27 +935,42 @@ func (b *Binance) SubmitOrder(ctx context.Context, s *order.Submit) (order.Submi
 		var oType string
 		switch s.Type {
 		case order.Limit:
-			oType = "LIMIT"
+			oType = cfuturesLimit
 		case order.Market:
-			oType = "MARKET"
+			oType = cfuturesMarket
 		case order.Stop:
-			oType = "STOP"
+			oType = cfuturesStop
 		case order.TakeProfit:
-			oType = "TAKE_PROFIT"
+			oType = cfuturesTakeProfit
 		case order.StopMarket:
-			oType = "STOP_MARKET"
+			oType = cfuturesStopMarket
 		case order.TakeProfitMarket:
-			oType = "TAKE_PROFIT_MARKET"
+			oType = cfuturesTakeProfitMarket
 		case order.TrailingStop:
-			oType = "TRAILING_STOP_MARKET"
+			oType = cfuturesTrailingStopMarket
 		default:
 			return submitOrderResponse, errors.New("invalid type, check api docs for updates")
 		}
-		o, err := b.FuturesNewOrder(ctx,
-			s.Pair, reqSide,
-			"", oType, "GTC", "",
-			s.ClientOrderID, "", "",
-			s.Amount, s.Price, 0, 0, 0, s.ReduceOnly)
+		o, err := b.FuturesNewOrder(
+			ctx,
+			&futuresNewOrderRequest{
+				symbol:           s.Pair,
+				side:             reqSide,
+				positionSide:     "",
+				orderType:        oType,
+				timeInForce:      "GTC",
+				newClientOrderID: s.ClientOrderID,
+				closePosition:    "",
+				workingType:      "",
+				newOrderRespType: "",
+				quantity:         s.Amount,
+				price:            s.Price,
+				stopPrice:        0,
+				activationPrice:  0,
+				callbackRate:     0,
```

Comment:
Now that its a request struct, we can remove all the default value assignments and have:
```
&futuresNewOrderRequest{
				symbol:           s.Pair,
				side:             reqSide,
				orderType:        oType,
				timeInForce:      "GTC",
				newClientOrderID: s.ClientOrderID,
				quantity:         s.Amount,
				price:            s.Price,
				reduceOnly: s.ReduceOnly,
}
```

Reply:
I like the "explicit is better than implicit" mantra, but OK. :}